package.json

```diff
@@ -1,16 +1,22 @@
 {
-  "name": "typically_bot",
+  "name": "bot",
   "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
+  "description": "The Discord companion made for you.",
+  "private": true,
```

Comment:
Do we need to mark this as private since, for one, this isn't a package that would be published on npm, and for two, it's in a public repository

Reply:
`private` ensures under no circumstances is it accidentally published. that is all it does.